code/modules/mob/mob.dm

```diff
@@ -487,6 +487,7 @@ var/list/slot_equipment_priority = list( \
 		return 0 //Unsupported slot
 		//END HUMAN
 
+// If you're looking for `reset_perspective`, that's a synonym for this proc.
```

Comment:
I mean, replacing it is all of 2 seconds, I just try to keep diffs as small as I can. 1 commit, coming right up!